src/covsonar/dbm.py

```diff
@@ -2442,15 +2441,18 @@ def create_profile_cases(
                         if match:
                             case, val = processing_funcs[mutation_type](match)
                             cases.append(
-                                f"CASE WHEN {' AND '.join(case)} THEN 1 ELSE 0 END AS mutation_{ids[mutation]}"
+                                f"SUM(CASE WHEN {' AND '.join(case)} THEN 1 ELSE 0 END) AS mutation_{ids[mutation]}"
```

Comment:
Why SUM? I think this is nor necessary. If the conditions are met we assign 1 else we assign 0. Makes the comparsion easier